go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/docker/buildx
 
-go 1.22.0
+go 1.23.0
```

Comment:
Was this intentional; i.e., go1.22 no longer supported with this change?

Reply:
This was intentional to be able to use `rangefunc`. After an offline conversation, I was informed that buildx has some downstream dependencies that haven't moved to go 1.23 yet and we generally try to make our `go.mod` files compatible with `stable - 1` go versions so we can introduce this usage when go 1.24 releases.

I've opened https://github.com/docker/buildx/pull/2978 to keep the bones of this functionality but to revert the compiler usage of the feature.